logger.js

```diff
@@ -12,10 +12,23 @@ function pinoLogger (opts, stream) {
   }
 
   opts = opts || {}
+
+  opts.customAttributeKeys = opts.customAttributeKeys || {}
+  var attributeKeys = {
+    req: opts.customAttributeKeys.req || 'req',
+    res: opts.customAttributeKeys.res || 'res',
+    err: opts.customAttributeKeys.err || 'err',
+    responseTime: opts.customAttributeKeys.responseTime || 'responseTime'
+  }
+  delete opts.customAttributeKeys
```

Comment:
Can you avoid the delete ?

Reply:
I added this to be consistent the other pino-http specific options, which are all `delete`-ed after consumption. My assumption was that this is to prevent them being passed into the wrapped pino instance. I can of course remove this if not required?

Reply:
Let's leave, not really required right now. This needs a bigger clean up.